src/toil/cwl/cwltoil.py

```diff
@@ -852,9 +866,9 @@ def makeJob(tool, jobobj, step_inputs, runtime_context, conditional):
                 if isinstance(r, string_types) and ("$(" in r or "${" in r):
                     # Found a dynamic resource requirement so use a job wrapper
                     job = CWLJobWrapper(tool, jobobj, runtime_context, conditional=conditional)
-                    return (job, job)
+                    return job, job
         job = CWLJob(tool, jobobj, runtime_context, conditional=conditional)
-        return (job, job)
+        return job, job
```

Comment:
This looks weird and there should be a comment justifying why we need to return the same thing twice.

Reply:
Above it returns `wfjob, followOn` when it's a workflow, and `job, job` otherwise.  I think it should probably be refactored, but will leave a comment for now.